src/blob/core.rs

```diff
@@ -191,8 +191,8 @@ impl Blob {
         self.index.get_entry(key, self.file.clone())
     }
 
-    async fn lookup(&self, key: &[u8], meta: Option<&Meta>) -> Option<Location> {
-        if self.contains(key) {
+    async fn get_location(&self, key: &[u8], meta: Option<&Meta>) -> Option<Location> {
```

Comment:
why did you change from lookup to get_location?

Reply:
No particular reason, thought it would be more clear